### PR TITLE
SParts labs proposal

### PR DESCRIPTION
**Pull request by MarkGisi**
Signed-off-by: markgisi <mark.gisi@windriver.com>

### CODE REVIEW

File: labs/SParts.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Lab Name
+
+SParts
+
+# Short Description
+
+The Software Parts (SParts) project delivers a Sawtooth-based ledger that provides both accountability and access to the open source components used in the construction of a software part. A software part is any software component (e.g., library, application, container or an entire operating system runtime) that is comprised of between 0% and 100% open source.
```
Comment (tkuhrt) [inline labs/SParts.md]: Change project to lab.

File: labs/SParts.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+The initial focus is to track the open source from which today's manufactured products and devices are constructed (think IoT). The project allows any organization, supply chain or community to easily spin up a distributed ledger that tracks: i) the open source components used and ii) their corresponding compliance artifacts (e.g., source code, notices, SPDX data, security vulnerability data, …) for the software parts used within a supply chain. A number of important benefits are obtained by knowing which open source components are used such as: 1) ensuring manufacturers are able to identify and secure the distribution rights (licenses) for all open source components; 2) understanding the impact of open source security vulnerabilities; 3) enable identification of cryptography technologies (e.g., FIPS 140-2 certification, export licensing); and 4) enable accurate reporting on all open source parts as a requirement to obtaining functional safety certification for safety critical products (e.g., medical devices, aircraft, autonomous vehicles, elevators, …). The distributable ledger provides both accountability over and access to the compliance artifacts for any given software part.
```
Comment (tkuhrt) [inline labs/SParts.md]: Change project to lab.

**Comment by tkuhrt** [comment]
LGTM +1

**Comment by vipinsun** [comment]
Looks clean + 1

**Comment by tkuhrt** [comment]
Created https://github.com/hyperledger-labs/SParts

**Comment by lehors** [comment]
Belated +1 :)

**Pull request merged**